requirements-dev.txt

```diff
@@ -1,9 +1,10 @@
-pre-commit
+pre-commit  # needed for pre-commit hooks in the git commit command
 
+# For the test suite
 pytest==7.2.0
-pytest-asyncio==0.20.3
+pytest-asyncio==0.20.3  # needed for async tests
```

Comment:
"needed because `pytest` doesn't come with native support for coroutines as tests"